src/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs

```diff
@@ -83,9 +89,11 @@ public AcquireTokenInteractiveParameterBuilder WithUseEmbeddedWebView(bool useEm
         // TODO: UIBehavior struct is INTERNAL on .net core...  (can we change that?)
```

Comment:
Yes we can, but it will be an empty struct on .net core, at least until we implement system browser auth.